xdebug.c

```diff
@@ -787,6 +793,7 @@ ZEND_DLEXPORT void xdebug_statement_call(zend_execute_data *frame)
 
 ZEND_DLEXPORT int xdebug_zend_startup(zend_extension *extension)
 {
+	xdebug_save_statement_handler(extension, xdebug_statement_call);
```

Comment:
When starting the extension, we save a pointer to it and to the statement handler function